misc/services/mobile-webui/mobile-webui-frontend/src/containers/activities/distribution/DistributionLinePickFromScreen.jsx

```diff
@@ -43,10 +45,16 @@ const DistributionLinePickFromScreen = () => {
   };
 
   return (
-    <BarcodeScannerComponent
+    <ScanHUAndGetQtyComponent
+      key={`${applicationId}_${wfProcessId}_${activityId}_${lineId}_scan`}
+      qtyTargetCaption={trl('general.QtyToPick')}
+      qtyCaption={trl('general.Qty')}
+      qtyTarget={qtyToPickRemaining}
+      uom={uom}
+      //
       resolveScannedBarcode={resolveScannedBarcode}
-      onResolvedResult={onBarcodeScanned}
-      continuousRunning={true}
+      onResult={onResult}
+      onClose={() => history.goBack()}
```

Comment:
why removing
```
continuousRunning={true}
````
?

Reply:
I've replace the BarcodeScannerComponent with ScanHUAndGetQtyComponent